recipes/libsystemd/all/conanfile.py

```diff
@@ -68,9 +69,17 @@ def requirements(self):
     def validate(self):
         if self.settings.os != "Linux":
             raise ConanInvalidConfiguration("Only Linux supported")
+        if os.getenv('NOT_ON_C3I', '0') == '0' and Version(self.version) >= "253.10" and self.settings.compiler == "gcc" and Version(self.settings.compiler.version) == "9":
```

Comment:
I would suggest using config instead, much safer than env vars, and be customized by package. It could be `user.c3i.skip:libsystemd=True`, so in case needing this workaround again (IIRC we already used it in the past) we just add a new package name, or a wild card like `*`.


https://docs.conan.io/2/reference/config_files/global_conf.html

Reply:
That sounds great to me!

Reply:
Done.